plugins/ProfessionalServices/templates/pluginAdvertising.twig

```diff
@@ -0,0 +1,27 @@
+<div
+    class="pluginPromo"
+    vue-entry="CoreHome.ContentBlock"
+    content-title="{{ title }}"
+    image-url="plugins/ProfessionalServices/images/{{ imageName|default('ad-' ~ plugin.name ~ '.png')|lower }}"
+    image-alt-text="{{ 'ProfessionalServices_AltTextPreviewImageFor'|translate(plugin.displayName) }}"
+>
+    <div class="promo-content">
+        {% if listOfFeatures|default([])|length %}
+            <div class="promo-features">
+                <ul>
+                    {% for feature in listOfFeatures %}
+                        <li><span class="promo-icon icon-ok"></span>{{ feature|raw }}</li>
+                    {% endfor %}
+                </ul>
+            </div>
+        {% endif %}
+    </div>
+    <div class="promo-actions">
+        <a class="btn" target="_blank" rel="noreferrer noopener" href="{{ plugin.shop.url|default('')|e('html_attr') }}">
+            {{ 'ProfessionalServices_CTAStartFreeTrial'|translate }}
+        </a>
+        <a class="learn-more" href="#" vue-directive="Marketplace.PluginName" vue-directive-value="{{ { pluginName: plugin.name }|json_encode }}" aria-label="{{ 'ProfessionalServices_CTALearnMore'|translate(plugin.displayName) }}">
```

Comment:
Just highlighting here that this is the only difference compared to the code for 5.x where the directive is called [`CorePluginsAdmin.PluginName`](https://github.com/matomo-org/matomo/pull/21402/files#diff-31afc88a22c4f772cefd35f5ab48267d2352e691864b10d6ae6e612bedd99dbcR23) and here for 4.x it's called `Marketplace.PluginName`.